factionboardlua.ttslua

```diff
@@ -44,6 +44,10 @@ function toggleButtons(player, value, id)
     self.UI.setAttribute(id..'Buttons', 'active', value)
 end
 
+function setColorRemote(color)
+   setColor(nil, color)
+end
```

Comment:
```suggestion
function setColorRemote(params)
   setColor(nil, params[1])
end
```

You can only call a method on another object with a table of arguments.

Reply:
That's not true, you can only call with *one* argument, it does not need to be a table